tree/dataframe/inc/ROOT/RDF/RLoopManager.hxx

```diff
@@ -85,9 +86,26 @@ public:
    }
 };
 
-} // ns RDF
-} // ns Internal
+class DatasetReaderKey {
```

Comment:
While this seems like a nice encapsulation / abstraction it also seems very much unused?

Reply:
see above :man_facepalming:

Reply:
removed and squashed the commits that removes it with the commit that introduced it